GameOfRunes/GameOfRunes/GameEngine/Components/ScoreComponent.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ScoreComponent.swift
+//  GameOfRunes
+//
+//  Created by Andy on 25/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+class ScoreComponent: GKComponent, Component {
+    var type: ComponentType {
+        .scoreComponent
+    }
+    
+    private var _scorePoints: Int
+    private let scoreNode: SKNode?
```

Comment:
help me add a TODO I will fix this portion for the player component

Reply:
Not sure exactly what you mean - I already refactored everything into PlayerComponent. Can see if I did it the way you intended to?

Reply:
Ok i'm gonna remove the scoreNode variable here

Reply:
yeah cuz not supposed to have the node here